layers/radiotap.go

```diff
@@ -42,6 +43,10 @@ const (
 	RadioTapPresentTxFlags
 	RadioTapPresentRtsRetries
 	RadioTapPresentDataRetries
+	_
+	RadioTapPresentMcs
```

Comment:
In general, prefer capitalizing acronyms everywhere... Not sure if MCS and VHT are acronyms, but if so, capitalizing them and other new vars/functions/etc. with acronyms in them would be great.  We don't do it as consistently as we like, but try to do it for new code.